mcc/odin/odin/snapshot_list_test.go

```diff
@@ -17,3 +21,39 @@ var exampleSnapshot1 = &rds.DBSnapshot{
 	MasterUsername:       aws.String("owner"),
 	Status:               aws.String("available"),
 }
+
+type listSnapshotsCase struct {
+	testCase
```

Comment:
This is another type, defined in `odin_test.go`, which provides expected return and expected error message, and the `check` method, which is a helper for testing everything is ok within the test.